packages/editor/package.json

```diff
@@ -91,10 +93,16 @@
     "@types/react": "^18.3.12",
     "@types/react-dom": "^18.3.1",
     "@types/ws": "~8.5.12",
+    "@typescript-eslint/eslint-plugin": "^8.12.2",
+    "@typescript-eslint/parser": "^8.12.2",
     "@vitejs/plugin-react": "^4.3.3",
     "@vitest/browser": "^2.1.4",
     "@xstate/react": "^4.1.3",
+    "babel-plugin-react-compiler": "beta",
     "dotenv": "^16.4.5",
+    "eslint": "8",
```

Comment:
Guess who's back 😂

Reply:
Well, it doesn't seem like biome does a good job validating useEffect deps? 😅